src/QuantEcon.jl

```diff
@@ -11,21 +11,22 @@ using DSP: TFFilter, freqz
 using Primes: primes
 using Compat: view, @compat
 
+using StatsBase
+import StatsBase:ecdf
```

Comment:
Let's also replace these two lines with

```Julia
using StatsBase: ecdf
```

This will make Julia only bring that one function from the StatsBase module into scope inside QuantEcon, helping us avoid cluttering up our namespace.